Error Building Provider ` no field or method Certificate`

I'm trying to build and test the provider locally. Following the quickstart guide in the README I'm receiving the following errors:

```
root@45fc21f24114:/main# make build
go mod tidy
==> Checking that code complies with gofmt requirements...
go install
# github.com/okta/terraform-provider-okta/okta
okta/data_source_okta_app_saml_metadata.go:67:57: desc.SSODescriptor.RoleDescriptor.KeyDescriptors[0].KeyInfo.Certificate undefined (type saml.KeyInfo has no field or method Certificate)
okta/resource_okta_app_saml.go:437:58: desc.SSODescriptor.RoleDescriptor.KeyDescriptors[0].KeyInfo.Certificate undefined 
(type saml.KeyInfo has no field or method Certificate)
okta/saml.go:45:52: desc.KeyInfo.Certificate undefined (type saml.KeyInfo has no field or method Certificate)
okta/saml.go:47:49: desc.KeyInfo.Certificate undefined (type saml.KeyInfo has no field or method Certificate)
make: *** [GNUmakefile:23: build] Error 2
```

Any insight would be great. Thanks in advance.

### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

--- a/okta/data_source_okta_app_saml_metadata.go
+++ b/okta/data_source_okta_app_saml_metadata.go
@@ -64,6 +64,6 @@ func dataSourceAppMetadataSamlRead(ctx context.Context, d *schema.ResourceData, 
 	syncSamlEndpointBinding(d, desc.SingleSignOnServices)
 	_ = d.Set("entity_id", metadataRoot.EntityID)
 	_ = d.Set("want_authn_requests_signed", desc.WantAuthnRequestsSigned)
-	_ = d.Set("certificate", desc.KeyDescriptors[0].KeyInfo.Certificate)
+	_ = d.Set("certificate", desc.KeyDescriptors[0].KeyInfo.X509Data.X509Certificates[0].Data)
 	return nil
 }

--- a/okta/resource_okta_app_saml.go
+++ b/okta/resource_okta_app_saml.go
@@ -434,7 +434,7 @@ func resourceAppSamlRead(ctx context.Context, d *schema.ResourceData, m interfac
 		key := getExternalID(uri, app.Settings.SignOn.IdpIssuer)
 		_ = d.Set("entity_url", uri)
 		_ = d.Set("entity_key", key)
-		_ = d.Set("certificate", desc.KeyDescriptors[0].KeyInfo.Certificate)
+		_ = d.Set("certificate", desc.KeyDescriptors[0].KeyInfo.X509Data.X509Certificates[0].Data)
 	}
 	appRead(d, app.Name, app.Status, app.SignOnMode, app.Label, app.Accessibility, app.Visibility, app.Settings.Notes)
 	if app.SignOnMode == "SAML_1_1" {

--- a/okta/saml.go
+++ b/okta/saml.go
@@ -42,9 +42,9 @@ func syncSamlCertificates(d *schema.ResourceData, descriptors []saml.KeyDescript
 	for _, desc := range descriptors {
 		switch desc.Use {
 		case "encryption":
-			_ = d.Set("encryption_certificate", desc.KeyInfo.Certificate)
+			_ = d.Set("encryption_certificate", desc.KeyInfo.X509Data.X509Certificates[0].Data)
 		case "signing":
-			_ = d.Set("signing_certificate", desc.KeyInfo.Certificate)
+			_ = d.Set("signing_certificate", desc.KeyInfo.X509Data.X509Certificates[0].Data)
 		}
 	}
 }

